game-core/src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -102,6 +107,35 @@ public static GameData loadGame(final InputStream is) throws IOException {
     }
   }
 
+  private static void promptToLoadNewerSaveGame(final Version version) throws IOException {
+    // we can still load it because our engine is compatible, however this save was made by a
+    // newer engine, so prompt the user to upgrade
+    // this is needed because a newer client might depend on variables that are part of the new
+    // save but will be stripped by the old client. When the old client re-saves the data, the
+    // new client will load the save game and be in odd state.
+    final String messageString =
+        "This save was made by a newer version of TripleA."
+            + "\nHowever, because the first version number is the same as your current version, "
```

Comment:
> nHowever, because the first version number is the same as your current version, "
            + "we can still open the savegame. 

This is a level of detail that we can probably omit. The fact that the game is offering to load the save at all already implies this level of compatibility.